modules/core/src/main/scala/lucuma/odb/api/model/ObservationModel.scala

```diff
@@ -9,6 +9,7 @@ import lucuma.core.`enum`.{ObsActiveStatus, ObsStatus}
 import lucuma.core.optics.syntax.lens._
 import lucuma.core.model.{Asterism, Observation, Program, Target}
 import cats.{Eq, Monad}
+import cats.Order.catsKernelOrderingForOrder
```

Comment:
I always need this but I'm never sure what the right way to import it is.

Reply:
Maybe cats.implicits.order?

Reply:
You can use `cats.implicits._` or `cats.implicits.catsKernelOrderingForOrder` I believe.